lib/src/api/opt/auth.rs

```diff
@@ -83,10 +83,36 @@ pub struct Scope<'a, P> {
 
 impl<T, P> Credentials<T, Jwt> for Scope<'_, P> where P: Serialize {}
 
-/// A JSON Web Token for authenticating with the server
+/// A JSON Web Token for authenticating with the server.
+///
+/// This struct represents a JSON Web Token (JWT) that can be used for authentication purposes.
+/// It is important to note that this implementation does not provide any security measures to
```

Comment:
The `Jwt` wrapper does provide some simple measures to protect against accidental leakage. When using the wrapper

- The debug implementation just prints `Jwt(REDACTED)`
- Display is not implemented so you can't call `.to_string()` on it

Tokens are sensitive data anyway so they should always be protected. It's the plain string exposed by `as_insecure_token` and `into_insecure_token` that users should worry about the most as it's easy to accidentally leak that one.